docker/migrate/src/main.ts

```diff
@@ -0,0 +1,64 @@
+import "process";
+import minimist from "minimist";
+import { buildIAMAuthenticationConnectionString } from "./rds.js"
```

Comment:
I have a minor preference for `esbuild --platform=node` over `tsc` because the bundling means you don't have to do ugly things like import from `*.js`. Not a big deal though.